virttest/qemu_monitor.py

```diff
@@ -2373,3 +2373,78 @@ def transaction(self, job_list):
         """
         transaction_args = {"actions": job_list}
         return self.cmd("transaction", transaction_args)
+
+    def add_dirty_bitmap(self, node, name, granularity=65536):
```

Comment:
methods add_dirty_bitmap(), remove_dirty_bitmap(), clear_dirty_bitmap seems to be similar, just curious to know do we really need to have separate methods.
Instead,

```
def dirty_bitmap(self, operation, node, name, granularity=65536):
    cmd = "block-dirty-bitmap-%s" % (operation)
    self.verify_supported_cmd(cmd)
    args = {"node": node, "name": name}
    if granularity:
        args["granularity"] = granularity
    return self.cmd(cmd, args)
```